src/main/resources/hudson/plugins/git/GitTagAction/tagForm.jelly

```diff
@@ -37,10 +37,106 @@ THE SOFTWARE.
         </j:forEach>
       </ul>
     </d:tag>
+    <d:tag name="blockWrapper">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div>
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <table class="middle-align">
+            <d:invokeBody/>
+          </table>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="row">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div class="tr">
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <tr>
+            <d:invokeBody/>
+          </tr>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="heading">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div class="th">
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <th>
+            <d:invokeBody/>
+          </th>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="td">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div class="td">
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <td>
+            <d:invokeBody/>
+          </td>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="td-colspan-3">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div class="td">
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <td colspan="3">
+            <d:invokeBody/>
+          </td>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="td-width-100">
+      <j:choose>
+        <j:when test="${divBasedFormLayout}">
+          <div class="td">
+            <d:invokeBody/>
+          </div>
+        </j:when>
+        <j:otherwise>
+          <td width="100%">
+            <d:invokeBody/>
+          </td>
+        </j:otherwise>
+      </j:choose>
+    </d:tag>
+    <d:tag name="td-nowrap">
```

Comment:
you could instead add a style one to the other td tag and just not pass it in when you don't need

For it to work in your IDE it's this I think:
`<st:attribute name="style"/>

and then `style="${attrs.style}"`

Reply:
Considering the low usage of this action, I intentionally left it simple without additional styling.  Let me know if you find an error.